man.Rmd

```diff
@@ -147,27 +190,49 @@ str_unique <- function(string, ...) {
 Here the introduction includes the title ("Remove duplicated strings") and a basic description of what the function does.
 The introduction is followed by five tags: two `@params`, one `@returns`, one `@seealso`, one `@examples`, and one `@export`.
 
-Note that each line of the block is 80 characters wide (matching the line length of the surrounding R code) and the second and subsequent lines of the long `@param` tag are indented, which makes the entire block easier to scan.
+Note that the block has an intentional line length (typically the same as that used for the surrounding R code) and the second and subsequent lines of the long `@param` tag are indented, which makes the entire block easier to scan.
 You can get more roxygen2 style advice in the [tidyverse style guide](https://style.tidyverse.org/documentation.html).
 
+::: callout-tip
+## RStudio
+
+It can be aggravating to manually manage the line length of roxygen comments, so be sure to try out *Code \> Reflow Comment* (Ctrl/Cmd+Shift+/).
+:::
+
 The following sections go into more depth for the most important tags.
 We start with the introduction, which provides the title, description, and details.
 Then we cover the inputs (the function arguments), outputs (the return value), and examples.
 Next we discuss links and cross-references, then finish off with techniques for sharing documentation between topics.
 
+### Key markdown features
+
+TODO: call out some useful markdown+roxygen features here.
```

Comment:
I haven't written this yet, but I think it would be useful.

Reply:
I think it's worth mentioning linking, but the rest is basic markdown that I don't think we need to repeat.

Reply:
I just pushed this section before I saw this. So there's something for you to react to now.

Reply:
I've got this little section, because I think it will actually push some useful info into people's brains. Let's see if we get any +/- feedback on it and how we feel about it later.